exts/cesium.omniverse/cesium/omniverse/ui/attributes/custom_attribute_widgets.py

```diff
@@ -0,0 +1,70 @@
+from typing import List
+import omni.ui as ui
+from pxr import Sdf
+from cesium.usd.plugins.CesiumUsdSchemas import (
+    RasterOverlay as CesiumRasterOverlay,
+)
+
+
+def _update_range(stage, prim_paths):
+    for path in prim_paths:
+        rasterOverlay = CesiumRasterOverlay.Get(stage, path)
+        attr = rasterOverlay.GetAlphaAttr()
+        current_value = attr.Get()
+        new_value = max(0, min(current_value, 1.0))
+        attr.Set(new_value)
```

Comment:
Is it possible to not hardcode `CesiumRasterOverlay`? I imagine we'll want to use this widget for other prim types.

Also it's not clear to me how the min/max options that are passed to `build_slider` are getting set. Seems like 0 and 1 are still hardcoded in a couple places.

Reply:
Ah, I'd left 0 and 1 in `widget_kwargs`. Gone now. 

I was able to remove the dependency on `CesiumRasterOverlay`. 

Note: There were issues operating on generic prims. For some reason, only class functions to access attributes would work (e.g., `GetAlphaAttr`) in event handlers. Fortunately, we don't need `update_range` anyways. It's cruft from an example where bounds and its associated attributes needed to be updated. We might possible run into this in the future if we want to do something like update bounds on generic prims.